eslint.common.config.js

```diff
@@ -0,0 +1,99 @@
+import jsdoc from "eslint-plugin-jsdoc";
+import ava from "eslint-plugin-ava";
+import globals from "globals";
+import js from "@eslint/js";
+import google from "eslint-config-google";
+
+export default [{
+	ignores: [ // Common ignore patterns across all tooling repos
+		"**/coverage/",
+		"test/tmp/",
+		"test/expected/",
+		"test/fixtures/",
+		"**/docs/",
+		"**/jsdocs/",
+	],
+}, js.configs.recommended, google, ava.configs["flat/recommended"], {
+	name: "Common ESLint config used for all tooling repos",
+
+	plugins: {
+		jsdoc,
+	},
+
+	languageOptions: {
+		globals: {
+			...globals.node,
+		},
+
+		ecmaVersion: 2023,
```

Comment:
why we allow ES2023 here but in the eslint.config.js only ES2022?

Reply:
This is the common eslint config that is shared accross the reops. It's just a copy.

Regarding the one in `eslint.config.js`, this is only for a certain file- `bin/ui5.cjs`.
It states that it must support as many NodeJS versions as possible, but I suppose we're limited to the engines deinfed in `package.json`.

And that's why they differ

Reply:
The idea with `ui5.cjs` was that you should still get the console message about using an unsupported Node.js version instead of having e.g. a SyntaxError before the message can even be showed.